src/api/localStorage.js

```diff
@@ -0,0 +1,110 @@
+export class LocalStorage {
+  constructor() {
+    this.users = JSON.parse(localStorage.getItem('UserAccounts')) || [];
+  }
+
+  createUser(user) {
+    this.updateUsers();
+    user.id = this.generateId();
+    this.users.push(user);
+    localStorage.setItem('UserAccounts', JSON.stringify(this.users));
```

Comment:
Since you are already setting and updating a this.users object, could you just add an updateDatabase function that sets the database to be the current this.users property?

Reply:
Not quite sure what we're talking about here. So for this function ^ I am pulling in the current  this.users with the this.updateUsers() to make sure I have the most up to date version of this.users since I have it setting in my constructor, then I am setting my id, pushing the new user with the new id to the current version of this.users and then setting that to the database. Do you mean like a helper that does the localStorage setting for me?

Reply:
Exactly. 

> Do you mean like a helper that does the localStorage setting for me?

Reply:
Complete